man/adoc/bpftrace.adoc

```diff
@@ -2610,7 +2610,7 @@ be rejected.
 interval:s:1 {
   $runqueues = (struct rq *)percpu_kaddr("runqueues", 0);
   if ($runqueues != 0) {         // The check is mandatory here
-    print($runqueues->nr_running);
+    print($runqueues->nr_queued);
```

Comment:
Same as for the tests - it would be good to use a different field here, if possible, so that the example works across different kernels.